recipe/meta.yaml

```diff
@@ -32,14 +32,16 @@ requirements:
     - packaging >=23.0
     - pathspec >=0.9.0
     - plumbum >=1.6.9
-    # TODO: remove when fixed upstream https://github.com/copier-org/copier/issues/1225
-    - pydantic >=1.10.2,<2
+    - pydantic >=2.0.3
     - pygments >=2.7.1
-    - python >=3.8
+    - python >=3.8,<4.0
     - pyyaml >=5.3.1
     - pyyaml-include >=1.2
     - questionary >=1.8.1
-    - git >=2.27
+    - typing-extensions >=3.7.4,<5.0.0
+
+    # questionary 2.0.0 has requirement prompt_toolkit<=3.0.36,>=2.0, but you have prompt-toolkit 3.0.39.
+    - prompt_toolkit <=3.0.36,>=2.0
```

Comment:
```suggestion
```

Could you open an PR to fix this upstream at https://github.com/conda-forge/questionary-feedstock
as we should avoid adding indirect dependency constraints to reduce maintenance and ease the env solver job.

Reply:
Fixed upstream by https://github.com/conda-forge/questionary-feedstock/pull/7